src/Pages/Login.js

```diff
@@ -47,15 +47,16 @@ const Login = () => {
     }
   };
 
+  function navigateToRegistration() {
+    window.location.href = "http://localhost:3000/registration"
```

Comment:
1. lets not use hardcoded URLs and change them to vars like  ${process.env.REACT_APP_BACKEND} that means we could change only 1 in option file and it will change everywhere in whole app.
2. can you also resolve the conflict and use function that already is implemented 
`handleSignUpBtn = () => {
    navigate('/registration')
  }`